Add rpath to find the libcairo-sphinx library to avoid this error while
running cairo-sphinx

root@cc-x5-2-5:~# /usr/bin/cairo-sphinx 
ld.so.1: cairo-sphinx: fatal: libcairo-sphinx.so: open failed: No such file or directory
ld.so.1: cairo-sphinx: fatal: relocation error: file /usr/bin/cairo-sphinx: symbol cairo_surface_create_similar: referenced symbol not found
Killed

--- a/util/cairo-sphinx/meson.build	2023-11-15 18:22:44.979320593 -0800
+++ b/util/cairo-sphinx/meson.build	2023-11-15 18:23:07.987908757 -0800
@@ -21,5 +21,6 @@
   dependencies: deps + [glib_dep, rt_dep, libcairo_dep, cairoboilerplate_dep, libcairoscript_dep],
   link_with: [libcairosphinx],
   link_args: extra_link_args,
+  install_rpath: libdir + ':' + join_paths(libdir,'cairo'),
   install: true,
 )
